rest/v8/index.ts

```diff
@@ -659,6 +660,18 @@ export const Routes = {
 	guildWelcomeScreen(guildID: Snowflake) {
 		return `/guilds/${guildID}/welcome-screen` as const;
 	},
+
+	/**
+	 * Route for:
+	 * - GET `/stage-instances/{channel.id}`
+	 * - POST `/stage-instances`
+	 * - PATCH `/stage-instances/{channel.id}`
+	 * - DELETE `/stage-instances/{channel.id}`
+	 */
+	stageInstance(channelID?: Snowflake) {
```

Comment:
This should be split into `stageInstances` and `stageInstance`, like `channelMessage(s)`